providers/class-two-factor-backup-codes.php

```diff
@@ -294,7 +298,7 @@ public function authentication_page( $user ) {
 	 * @return boolean
 	 */
 	public function validate_authentication( $user ) {
-		return $this->validate_code( $user, $_POST['two-factor-backup-code'] );
+		return $this->validate_code( $user, filter_input( INPUT_POST, 'two-factor-backup-code', FILTER_SANITIZE_NUMBER_INT ) );
```

Comment:
The backup codes could include alpha-numeric characters if the defaults for this helper are changed at some point:

https://github.com/WordPress/two-factor/blob/e380d6a92846d60647e6195184d9948e4235fd95/providers/class.two-factor-provider.php#L86-L96

Reply:
Fixed.